command_server/controllers/main.py

```diff
@@ -144,61 +157,57 @@ def set_invaders(self):
-        if action == 'bullet_invader':
-            move_bullet = True
-            while move_bullet:
-                self.Invaders[unit_number].move_bullet()
-                if self.Invaders[unit_number].check_fire(self.hero, self.game_field['height']):
+    def bullet_invader(self, unit_number=0):
+        move_bullet = True
+        while move_bullet:
+            self.Invaders[unit_number].move_bullet()
+            if self.Invaders[unit_number].check_fire(self.hero, self.game_field['height']):
+                move_bullet = False
+                self.Invaders[unit_number].bullet = Bullet(self.Invaders[unit_number], self.Invaders[unit_number].bullet.image_filename)
+
+    def bullet_hero(self):
+        move_bullet = True
+        while move_bullet:
+            self.hero.move_bullet()
+            for _invader in self.Invaders:
+                if self.hero.check_fire(_invader, self.game_field['height']):
                     move_bullet = False
-                    self.Invaders[unit_number].bullet = Bullet(self.Invaders[unit_number], self.Invaders[unit_number].bullet.image_filename)
-        if action == 'bullet_hero':
-            move_bullet = True
-            while move_bullet:
-                self.hero.move_bullet()
-                # print 'bullet - ', game_object.hero.bullet.__dict__
-                for _invader in self.Invaders:
-                    if self.hero.check_fire(_invader, self.game_field['height']):
-                        move_bullet = False
-                        self.hero.bullet = Bullet(self.hero, self.hero.bullet.image_filename)
-        if action == 'move_right':
-            self.hero.move_to(self.hero.x + self.hero.speed, self.hero.y)
-        if action == 'move_left':
-            self.hero.move_to(self.hero.x - self.hero.speed, self.hero.y)
-
-    def run(self):
-        game_object = GameController(50, 50, 2)
-
-        '''this code for moving invaders. Work as a job.
-            We set moving_speed for positive - if reach the left coordinate of our game field
-            or negative  - if we reach the right coordinate of our game field '''
-
-        while not game_object.hero.is_dead and len(game_object.Invaders):
-            game_object.Invaders[0].set_speed(game_object.game_field['width'])
-            game_object.Invaders[-1].set_speed(game_object.game_field['width'])
-            check_if_move_y = game_object.Invaders[0].check_if_move_y()
-            random_number = randint(0, len(game_object.Invaders) - 1)
-            game_object.get_action('bullet_invader', random_number)
-            for invader in game_object.Invaders:
-                new_x = invader.x + invader.moving_speed
-                new_y = check_if_move_y and invader.y + invader.speed or invader.y
-                invader.move_to(new_x, new_y)
-
-            # for invader in game_object.Invaders:
-            #     print invader.__dict__
-
-            time.sleep(3)
-
-if __name__ == "__main__":
```

Comment:
In scope of main we have to initialize GameController() and then call game.run(). That's it